webpack.conf.js

```diff
@@ -0,0 +1,26 @@
+var webpack = require('webpack');
+
+module.exports = {
+    devtool: "inline-source-map",
```

Comment:
It'd be cool to (some day) have two bundles for production, one that includes sourcemaps and one that doesn't, that you could switch between using a query parameter or something. It'd make debugging live stores a lot easier. Not really in the scope of this PR, though.